dolphinscheduler-alert/src/test/java/org/apache/dolphinscheduler/alert/plugin/DolphinPluginLoaderTest.java

```diff
@@ -55,6 +48,6 @@ public void testLoadPlugins() throws Exception {
             throw new RuntimeException("load Alert Plugin Failed !", e);
         }
 
-        Assert.assertNotNull(alertPluginManager.getAlertChannelFactoryMap().get("email alert"));
+        Assert.assertNotNull(alertPluginManager.getAlertChannelFactoryMap().get("Email"));
```

Comment:
Avoid the magic number `Email`

Reply:
The alert module does not depend on specific plug-in modules, so defining constants here may not be a good solution.